test/blocks/section-metadata/section-meta.test.js

```diff
@@ -70,6 +70,16 @@ describe('Section Metdata', () => {
     expect(main.firstElementChild).to.be.eql(sec);
   });
 
+  it('add promobar behaviour to section', async () => {
+    const main = document.querySelector('main');
+    const sec = document.querySelector('.section.sticky-bottom .promobar').closest('.section');
+    const sm = sec.querySelector('.section-metadata');
+    await init(sm);
+    console.log(main.lastElementChild);
```

Comment:
This pollutes the test runner console, it should be removed.

Reply:
@overmyheadandbody I have made the requested changes.